tools/yarn-cling/lib/index.ts

```diff
@@ -31,7 +31,7 @@ export async function generateShrinkwrap(options: ShrinkwrapOptions): Promise<Pa
   const packageJsonDir = path.dirname(packageJsonFile);
 
   const yarnLockLoc = await findYarnLock(packageJsonDir);
-  const yarnLock: YarnLock = lockfile.parse(await fs.readFile(yarnLockLoc, { encoding: 'utf-8' }));
+  const yarnLock: YarnLock = lockfile.parse(await fs.readFile(yarnLockLoc, { encoding: 'utf8' }));
```

Comment:
Yuck. Someone missed the ball on strong typing here.

Reply:
I think this might be some stuff in `@types/node` somehow becoming a little stricter than before (possibly due to changes in newer versions of `node`).